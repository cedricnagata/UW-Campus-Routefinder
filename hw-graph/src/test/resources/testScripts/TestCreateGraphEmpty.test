# Tests that an empty Graph is created

# Create a graph
CreateGraph Graph

# Print the empty list of nodes in the graph
ListNodes Graph