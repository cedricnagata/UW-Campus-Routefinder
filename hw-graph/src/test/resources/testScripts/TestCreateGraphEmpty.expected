# Tests that an empty Graph is created

# Create a graph
created graph Graph

# Print the empty list of nodes in the graph
Graph contains: